test_suites/test_suite.rb

```diff
@@ -1,8 +1,13 @@
 require 'watir'
 require 'webdrivers'
-require_relative 'testcases'
 require_relative 'auxiliary_methods'
+require_relative 'tc_01'
```

Comment:
Try to use wildcards to require all tc files with pattern like this `tc_*`
[Related info](https://www.thoughtco.com/using-glob-with-directories-2907832)

Reply:
fixed